### PR TITLE
Add keyboard 4

### PR DESCRIPTION
**Pull request by JorisMonnet**

### CODE REVIEW

**Review by marluxiaboss — APPROVED** [review]
Warnings should be corrected

**Review by kapapp — APPROVED** [review]
Consider switching hardcoded strings to variables

File: app/src/androidTest/java/ch/epfl/sdp/blindwar/data/SpotifyResponseTest.kt [hunk 1]
```diff
@@ -61,16 +63,20 @@ class SpotifyResponseTest {
...
+                "ARTIST_URL",
+                "ARTIST_ID",
+                arrayListOf(spotifyImage),
+                "ARTIST_NAME",
+                10,
+                "ARTIST_TYPE",
+                "ARTIST_URI"
+            )
+        )
     }
```
Comment (Marty99-dev) [inline app/src/androidTest/java/ch/epfl/sdp/blindwar/data/SpotifyResponseTest.kt:80]: Some hardcoded values

File: app/src/main/java/ch/epfl/sdp/blindwar/ui/LoginActivity.kt [hunk 2]
```diff
@@ -19,7 +19,8 @@ class LoginActivity : AppCompatActivity() {
     fun loginButton(view: View) {
         val database = UserDatabase()
         val appStatistics = AppStatistics()
-        val user = User("Jojo@BlindWar.ch", appStatistics,"JoestAAr", "jojo", "star", 123123213, null)
+        val user =
+            User("Jojo@BlindWar.ch", appStatistics, "JoestAAr", "jojo", "star", 123123213, null)
```
Comment (Marty99-dev) [inline app/src/main/java/ch/epfl/sdp/blindwar/ui/LoginActivity.kt:23]: We must change that next week !

**Review by Marty99-dev — APPROVED** [review]

**Pull request merged**